src/main/java/com/cgdp/library/dto/AuthorDTO.java

```diff
@@ -0,0 +1,14 @@
+package com.cgdp.library.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@AllArgsConstructor
+@Getter
+@Setter
+public class AuthorDTO {
```

Comment:
Can be converted to a [record](https://www.baeldung.com/java-record-keyword), and annotationd won't be necessary
```suggestion
public record AuthorDTO(Long id, String name) {
```